python/ray/tests/test_actor.py

```diff
@@ -1274,37 +1272,16 @@ def locations_to_intervals_for_many_tasks():
 
     # Run a bunch of GPU tasks.
     locations_to_intervals = locations_to_intervals_for_many_tasks()
-    # Make sure that all but one of the GPUs were used.
-    assert (len(locations_to_intervals) == num_nodes * num_gpus_per_raylet - 1)
     # For each GPU, verify that the set of tasks that used this specific
     # GPU did not overlap in time.
     for locations in locations_to_intervals:
         check_intervals_non_overlapping(locations_to_intervals[locations])
     # Make sure that the actor's GPU was not used.
     assert actor_location not in locations_to_intervals
 
-    # Create several more actors that use GPUs.
-    actors = [Actor1.remote() for _ in range(3)]
-    actor_locations = ray.get(
-        [actor.get_location_and_ids.remote() for actor in actors])
-
-    # Run a bunch of GPU tasks.
-    locations_to_intervals = locations_to_intervals_for_many_tasks()
-    # Make sure that all but 11 of the GPUs were used.
-    assert (
```

Comment:
These asserts are flaky because they assume that all of the tasks run concurrently, which may not be true if workers take a while to start up or something is really slow.